SophiApp/SophiApp/Localizations/EN.xaml

```diff
@@ -4,23 +4,23 @@
     xmlns:system="clr-namespace:System;assembly=mscorlib">
 
     <system:String x:Key="Conditions.LoggedUserAdmin">The logged-on user doesn't have admin rights</system:String>
-    <system:String x:Key="Conditions.NoNewVersion">A new version found</system:String>
+    <system:String x:Key="Conditions.NoNewVersion">A new version was found</system:String>
     <system:String x:Key="Conditions.OsBuildVersion">Supported Windows 10 versions: 2004/20H2/21H1/21H2</system:String>
-    <system:String x:Key="Conditions.OsIsInfected">Probably your OS was infected via the Win 10 Tweaker backdoor</system:String>
+    <system:String x:Key="Conditions.OsIsInfected">Your OS may have been infected through a backdoor in Win 10 Tweaker.</system:String>
```

Comment:
Can you elaborate a bit on this backdoor? I tried googling but got mostly Russian results.